_topics/en/covenants.md

```diff
@@ -75,9 +75,15 @@ see_also:
   - title: Vaults
     link: topic vaults
 ---
+> "In the context of Bitcoin, the most useful definition of
+> covenant is that it's when the scriptPubKey of a UTXO restricts the
+> scriptPubKey in the output(s) of a tx spending that utxo."  ---Anthony
```

Comment:
I assume that's a direct quote, but it's a bit odd that UTXO is capitalized once and lowercased once.

```suggestion
> covenant is that it's when the scriptPubKey of a UTXO restricts the
> scriptPubKey in the output(s) of a tx spending that UTXO."  ---Anthony
```

Reply:
Whoops, my fault, not AJs.  I uppercased one instance and missed the second.  Fixed, thanks!